modules/virt-features-for-storage-matrix.adoc

```diff
@@ -31,3 +31,10 @@
 1. PVCs must request a ReadWriteMany access mode
 2. PVCs must request a ReadWriteOnce access mode
 --
+
+[NOTE]
+====
+Virtual machines using a storage class with ReadWriteOnce (RWO) access mode or passthrough features such as
```

Comment:
s/using/that use

For readability, I'd suggest something like the following:

```
[NOTE]
====
You cannot live migrate virtual machines that use:

* A storage class with ReadWriteOnce (RWO) access mode
* Passthrough features such as SRI-OV and GPU

Do not set the `evictionStrategy` field to `LiveMigrate` for these virtual machines.
====
```

Reply:
Edited per your suggestion. Thanks.